### PR TITLE
r_gbif_checklist: some fixes

### PR DESCRIPTION
**Pull request by florisvdh**
## Description

Notably, this updates the date format; currently renders as: 

afbeelding

Plus some extra fixes.

I did the edit online in the Rmd file. @damianooldoni can you knit to md and add the extra commit please?

## Previewing the pull request

Thanks to GitHub Actions, an artifact (=zip file) of the rendered website is automatically created for each pull request.
This provides a way to preview how these updates will look on the website, useful to contributors and reviewers.

### Instructions to preview the updated website

1) On the PR page, you can find a "details" link under "checks - On PR, build the site and ...". Go there, click on the top link in the left sidebar ("Summary"), and download the generated artifact at the bottom of the page.
2) Decompress it and make sure the target directory is called 'tutorials' (you may need to rename it)
3) From the parent directory (just above the `tutorials` folder you created/renamed), run `python -m http.server 8887`, _or_ launch the Google Chrome [Web Server app](https://chrome.google.com/webstore/detail/web-server-for-chrome/ofhbbkphhbklhfoeikjpcbhemlocgigb) and point it at the parent directory.
4) Point your browser to http://localhost:8887/tutorials.
5) Review the updated website. As a contributor, you can push extra commits to update the PR. As a reviewer, you can accept/refuse/comment the PR.

**Note: for step 3, you can use any other simple HTTP server to serve the current directory if you don't have a Python 3 environment or Google Chrome available.**

### CODE REVIEW

**Comment by damianooldoni** [comment]
Thanks @florisvdh! I got also some nice inputs from @peterdesmet. I will add all of them in this PR and then I will knit it.
Thank you all 👍

**Comment by damianooldoni** [comment]
Copy pasting the improvements proposed by @peterdesmet for better tracking while committing them:
1. ik zou ook taxonID vermelden
2. done by GBIF automatically behind the scenes while when publishing the checklist.
3. Bij je nerdy touch is er een numbered list die niet mooi rendert, wrsch door ontbrekende lege lijn tussen paragraaf en punt “1. taxonomic information:”

De titel is ook iets nauwer in scope dan wat je uitlegt, dus misschien even hinten in de description

Titel: Get species checklist data from GBIF via with rgbif
Description: How to use rgbif function name_usage() to get species checklist data from GBIF as well as some other rgbif tips.

**Review by damianooldoni — APPROVED** [review]
Changes applied.

**Comment by florisvdh** [comment]
Looks OK, thanks!

@peterdesmet, if you want to re-review, then it's up to you to merge.

BTW: JSON and XML render nicely out of the box in Firefox:

afbeelding

File: content/tutorials/r_gbif_checklist/index.Rmd [hunk 1]
```diff
@@ -51,7 +51,7 @@ You get a list of two:
 names(red_list)
 ```
 
-the slot ´meta `contains metadata:
+The slot ´meta `contains metadata:
```
Comment (peterdesmet) [inline content/tutorials/r_gbif_checklist/index.Rmd:54]:
```suggestion
The slot `meta` contains metadata:
```

File: content/tutorials/r_gbif_checklist/index.Rmd [hunk 2]
```diff
@@ -79,8 +79,9 @@ kable(red_list$data[1:10,])
 
 But what do those keys stay for?
 
-- `key`, sometimes called `taxonKey`, is the identifier of the taxon within our checklist
-- `nubKey` is the identifier of the taxon within the [GBIF Backbone Taxonomy](https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c) dataset, or simply _backbone_. The link from the taxon in the red list checklist dataset the taxon in the backbone is done by GBIF automatically behind the scenes while publishing the checklist. You can control the match before publication of course, by using the gbif match utility in rgbif (see the vignette [match and download a long species list](https://docs.ropensci.org/rgbif/articles/downloading_a_long_species_list.html)) or the GBIF [data validator](https://www.gbif.org/tools/data-validator) web tool.
+- `key`, sometimes called `taxonKey`, is the GBIF key of the taxon. It is unique among all taxa published on GBIF.
```
Comment (peterdesmet) [inline content/tutorials/r_gbif_checklist/index.Rmd:82]:
```suggestion
- `key` (sometimes called `taxonKey`) is the taxon identifier assigned by GBIF. The same taxon in different checklists will have a different `key`.
```

File: content/tutorials/r_gbif_checklist/index.Rmd [hunk 3]
```diff
@@ -79,8 +79,9 @@ kable(red_list$data[1:10,])
...
+- `taxonID` is the identifier of the taxon within the red list checklist as provided by the authors
```
Comment (peterdesmet) [inline content/tutorials/r_gbif_checklist/index.Rmd:83]:
```suggestion
- `taxonID` is the taxon identifier as provided by the authors. The authors should keep it stable between publications.
```

File: content/tutorials/r_gbif_checklist/index.Rmd [hunk 4]
```diff
@@ -79,8 +79,9 @@ kable(red_list$data[1:10,])
...
+- `nubKey` is the identifier of the taxon within the [GBIF Backbone Taxonomy](https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c) dataset, or simply _backbone_. The link from the taxon in the red list checklist dataset to the taxon in the backbone is done by GBIF automatically behind the scenes when publishing the checklist. You can control the match before publication of course, by using the gbif match utility in rgbif (see the vignette [match and download a long species list](https://docs.ropensci.org/rgbif/articles/downloading_a_long_species_list.html)) or the GBIF [data validator](https://www.gbif.org/tools/data-validator) web tool.
```
Comment (peterdesmet) [inline content/tutorials/r_gbif_checklist/index.Rmd:84]:
```suggestion
- `nubKey` is the taxon identifier within the [GBIF Backbone Taxonomy](https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c) dataset, or simply _backbone_. The same taxon in different checklists will have the same `numKey`. The link between the taxon in the checklist and the taxon in the backbone is automatically created by GBIF behind the scenes when publishing a checklist. You can control the match before publication of course, by using the gbif match utility in rgbif (see the vignette [match and download a long species list](https://docs.ropensci.org/rgbif/articles/downloading_a_long_species_list.html)) or the GBIF [data validator](https://www.gbif.org/tools/data-validator) web tool.
```

**Review by peterdesmet — APPROVED** [review]
Suggested some minor rephrasing, but haven't committed or knitted. @damianooldoni can you do this and merge?

File: content/tutorials/r_gbif_checklist/index.Rmd (hunk 2 above)
Comment (damianooldoni) [reply content/tutorials/r_gbif_checklist/index.Rmd:82]: Nice rephrasing, thanks.

**Pull request merged**